src/styles.css

```diff
@@ -102,3 +107,69 @@ body {
   width: 50px;
   height: 50px;
 }
+.organism-description {
+  font-weight: bold;
+}
+.new-game-popup-container {
+  position: fixed;
```

Comment:
Avoid using `position: fixed` and `position: absolute`, because it can complicate your layout a lot.
Centering it on the screen can be achieved with flexbox using `justify-content` and `align-items`.